internal/nexodus/wireguard/route_darwin.go

```diff
@@ -0,0 +1,90 @@
+//go:build darwin
```

Comment:
how did you move these files? git seems to think the files were deleted and then created new. We need git to know that it's a move, not a delete/create.  Try `git mv`.